app/views/content/index.html.erb

```diff
@@ -113,7 +113,7 @@
                   <%= t.cell item.document_type %>
                   <%= t.cell number_with_delimiter(item.upviews, delimiter: ','), format: 'numeric' %>
                   <%= t.cell item.user_satisfaction %>
-                  <%= t.cell item.searches, format: 'numeric' %>
+                  <%= t.cell number_with_delimiter(item.searches, delimiter: ','), format: 'numeric' %>
```

Comment:
Can this go in the presenter?

Reply:
(Just for consistency with rest of the code)

Reply:
Sure, I've moved that, and the above similar line out of the presenter here b369d65aacf72f6f319b8a919655c7bbb26bad70